OJP/OJP_Fare.xsd

```diff
@@ -124,6 +124,11 @@
 					<xs:documentation>Problems related to this FARE result.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="FareEstimated" type="xs:boolean" default="true" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Is set to true, when the fare is only estimated by the service. If not set, it should be assumed to be true.</xs:documentation>
```

Comment:
1. Proposed documentation: "Whether the fare is only estimated by the service. Default is TRUE."
2. I wonder whether setting default as true is sensible, isn't the "usual" case rather it is calculated? What does "estimated" mean, how to react to this as a receiver?